src/test/java/teammates/test/cases/browsertests/InstructorCourseEnrollPageUiTest.java

```diff
@@ -90,7 +90,7 @@ private void testEnrollAction() throws Exception {
         InstructorCourseEnrollResultPage resultsPage = enrollPage.enroll(enrollString);
 
         // This is the full HTML verification for Instructor Course Enroll Results Page, the rest can all be verifyMainHtml
-        resultsPage.verifyHtml("/instructorCourseEnrollPageResult.html");
+        resultsPage.verifyHtml("/instructorCourseEnrollFirstPageResult.html");
```

Comment:
Are there better names for the 2 pages beside "First" and "Second"?

Reply:
I think it's okay for the names to be long, since they are already pretty long and some other HTML file names are pretty long too. Can just be sth like `...AllFields` or `...EmptySection`